recipes/types-xmltodict/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "types-xmltodict" %}
+{% set version = "0.14.0.20241009" %}
+{% set python_min = "3.8" %}
```

Comment:
Note that conda-forge does not support Python 3.8, while it is possible to keep this and the package will be installable there, there won't be any py38 updates. We recommend to remove this line and let conda-forge default min Python (3.9) be used instead. However, if you really need 3.8, keep it but knowing we won't support any issues that may rise.